aldryn_faq/locale/de/LC_MESSAGES/django.po

```diff
@@ -1,292 +1,301 @@
 # SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
-#, fuzzy
+# 
+# Translators:
+# Translators:
+# Angelo Dini <finalangeljp@hotmail.com>, 2016
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: Aldryn FAQ\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2016-03-17 12:02+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"POT-Creation-Date: 2016-06-20 11:22+0200\n"
+"PO-Revision-Date: 2016-06-20 09:45+0000\n"
+"Last-Translator: Angelo Dini <finalangeljp@hotmail.com>\n"
+"Language-Team: German (http://www.transifex.com/divio/aldryn-faq/language/de/)\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Language: de\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: admin.py:36
+#: admin.py:37
 msgid "Language Independent Fields"
-msgstr ""
+msgstr "Sprach unabhängige Felder"
 
-#: admin.py:39
+#: admin.py:40
 msgid "Advanced options"
-msgstr ""
+msgstr "Erweiterte Einstellungen"
 
 #: boilerplates/bootstrap3/templates/aldryn_faq/includes/question_list.html:41
-#: models.py:306
+#: models.py:306 templates/aldryn_faq/includes/question_list.html:33
 msgid "Category"
-msgstr ""
+msgstr "Kategorie"
 
 #: boilerplates/bootstrap3/templates/aldryn_faq/includes/question_list.html:45
 #: boilerplates/bootstrap3/templates/aldryn_faq/includes/ungrouped_question_list.html:26
-#: boilerplates/bootstrap3/templates/aldryn_faq/plugins/category_list.html:15
+#: boilerplates/bootstrap3/templates/aldryn_faq/plugins/category_list.html:12
 #: boilerplates/legacy/templates/aldryn_faq/includes/question_groups.html:13
 #: boilerplates/legacy/templates/aldryn_faq/includes/question_list.html:27
 #: boilerplates/legacy/templates/aldryn_faq/plugins/category_list.html:8
+#: templates/aldryn_faq/includes/question_list.html:28
+#: templates/aldryn_faq/includes/question_list.html:35
+#: templates/aldryn_faq/includes/ungrouped_question_list.html:19
+#: templates/aldryn_faq/plugins/category_list.html:17
 msgid "No entry found."
```

Comment:
Why do some entries have a full-stop, like this, and others don't? Is there a rationale?